client/components/domains/domain-product-price/index.jsx

```diff
@@ -40,11 +46,23 @@ class DomainProductPrice extends React.Component {
 		);
 	}
 
+	renderFormattedPrice() {
+		const { price, currencyCode } = this.props;
+		const rawPrice = replace( price, /[^0-9\.]/g, '' );
```

Comment:
I'd strongly advise against doing this logic yourself - the assumption that `.` is the delimited always is one I've been bitten by myself: https://github.com/Automattic/wp-calypso/pull/15611 :)
That PR also shows how I've handled that without resorting to parsing the formatted the price like this - which is a brittle approach because, the format can change (what if the `,` becomes the delimiter - which it _is_ for some currencies)?
Better operate on a raw price provided by the API - it's in the products list and should be easily passed down to this component.

Reply:
Totally agree with Igor here. We can even add raw data from the back-end if needed.